build.sbt

```diff
@@ -441,23 +435,42 @@ lazy val `scio-core`: Project = project
       "org.apache.beam" % "beam-vendor-guava-26_0-jre" % beamVendorVersion,
       "org.apache.commons" % "commons-compress" % commonsCompressVersion,
       "org.apache.commons" % "commons-math3" % commonsMath3Version,
-      "org.scalatest" %% "scalatest" % scalatestVersion % Test,
-      "org.slf4j" % "slf4j-api" % slf4jVersion,
-      "org.typelevel" %% "algebra" % algebraVersion,
-      "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion
-    ).map(_.withDottyCompat(scalaVersion.value)),
+      "org.slf4j" % "slf4j-api" % slf4jVersion
+    ),
+    // Scala dependencies to cross compile
```

Comment:
If I understand things correctly, we don't actually cross-compile the dependencies, we use the same dependencies in a cross-compilation setting.

Reply:
Indeed, it is *cross-version dependence*... We need better wording

Reply:
The new wording ("Scala dependencies available for 2.12, 2.13 and 3") is clearer :+1: